static/databrowser.html

```diff
@@ -5,10 +5,25 @@
     <script type="text/javascript" src="/common/js/mashlib.min.js"></script>
     <script>
       const $SOLID_GLOBAL_config = {}
-      document.addEventListener('DOMContentLoaded', function () {
+      document.addEventListener('DOMContentLoaded', async function () {
         const panes = require('mashlib')
         const UI = panes.UI
 
+        try {
+          var session = await SolidAuthClient.currentSession();
+          if (session && session.webId) {
+            // Check, that session has access to user data and it wasn't expired
+            var url = new URL(session.webId);
+            url.pathname = '/settings/privateTypeIndex.ttl';
```

Comment:
Good idea, though I would be hesitant to use `/settings/privateTypeIndex.ttl`. The spec doesn't reference that it MUST exist. The only reference is in a proposal written 3 years ago (https://github.com/solid/solid/blob/a25b6d11477fc3ec11d061c4945ff1c63e1ee65b/proposals/data-discovery.md)

But, databrowser.html does only exist on NSS, so I'll bring in @dmitrizagidulin for the final word on this